tachyon/math/finite_fields/cubic_extension_field_unittest.cc

```diff
@@ -147,7 +147,7 @@ TEST_F(CubicExtensionFieldTest, MultiplicativeOperators) {
     GF7_3 tmp = test.a;
     tmp *= test.b;
     EXPECT_EQ(tmp, test.mul);
-    tmp /= test.b;
+    ASSERT_TRUE(tmp /= test.b);
```

Comment:
I'm not sure if we can always assure the quotient to have value.

Reply:
There is only one test case run and the quotient has a concrete value!